hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -373,7 +372,21 @@ public boolean getIsNamespaceEnabled(TracingContext tracingContext)
           + " getAcl server call", e);
     }
 
-    isNamespaceEnabled = Trilean.getTrilean(NamespaceUtil.isNamespaceEnabled(client, tracingContext));
+    try {
+      LOG.debug("Get root ACL status");
```

Comment:
Why are we making the change of removing the NamespaceUtil class ? In my view we should not increase the diff

Reply:
This class was actually introduced in CPK pr: https://github.com/apache/hadoop/pull/6221/files#diff-aed851febeaa85ad9b0c254c00ebd21200b84d36bc99255a344db89319aab0b0.

Reason for adding this class was that we wanted to check namespace information in abfsClient and store. This is no more required, as we would need the information in store.

Reason for removing this class is to bring in the state which was before-cpk merge.